measuring-client/src/main/res/values/strings.xml

```diff
@@ -98,8 +100,6 @@
   <string name="toast_no_locations_found">No locations found</string>
   <string name="toast_measurement_deletion_success">Copies of the measurements on the server are not affected.</string>
   <string name="toast_item_deletion_success">The selected items were deleted.</string>
-  <string name="toast_measurement_paused">Trip paused</string>
-  <string name="toast_measurement_resumed">Trip resumed</string>
```

Comment:
these stings were located in the wrong section, i.e. this change is just a documentation typo fix